### DISCUSSION TITLE
Macs per IP

### DISCUSSION THREAD

**Discussion by Carlos-mb** [Security-Onion-Solutions/securityonion#13897] [Unsupported Versions]
Hi everyone,

I've been trying for some time to figure out how to generate a report that lists IPs by MAC address, or vice versa—MAC addresses by IP.

I've attempted to do this in both Dashboard and Kibana, but my current knowledge hasn't been enough to accomplish it.

The reason I want this report is to identify any IPs associated with multiple MAC addresses. This way, I can pinpoint which devices may be sharing their internet connection with others on my network.

I'd appreciate any ideas, references, or advice!

Thanks!

**Comment by TOoSmOotH** [comment]
You would need DHCP logs. Its the only reliable way to get MACs. Once the traffic is routed the MAC is no longer that of the initial client but of the router.

**Reply by Carlos-mb** [reply]
I agree with you, thats the reason why I want to get the MAC list by IP from S.O. logs, because I want to identify devices acting as routers.
